addons/l10n_in_edi/models/account_move.py

```diff
@@ -61,3 +61,12 @@ def _l10n_in_edi_is_managing_invoice_negative_lines_allowed(self):
         """
         param_name = 'l10n_in_edi.manage_invoice_negative_lines'
         return bool(self.env['ir.config_parameter'].sudo().get_param(param_name))
+
+    def action_view_einvoice(self):
+        if not self._get_l10n_in_edi_response_json():
+            raise UserError(_("There is no Indian E-invoice found."))
```

Comment:
Isn't this guarding against the case where `raise werkzeug.exceptions.Forbidden()`?